packages/datetime/test/common/dateUtilsTests.tsx

```diff
@@ -221,4 +221,124 @@ describe("dateUtils", () => {
             assertTimeIs(time, 11, 20);
         });
     });
+
+    describe("convert24HourMeridiem", () => {
+        it("returns given hour, if hour is AM and toAm", () => {
+            for (let hour = 0; hour <= 11; hour++) {
+                expect(DateUtils.convert24HourMeridiem(hour, true)).to.equal(hour);
+            }
+        });
+
+        it("returns given hour in PM, if hour is AM and not toAm", () => {
+            const pmHours = [];
+            for (let i = 12; i <= 23; i++) {
+                pmHours.push(i);
+            }
+            for (let amHour = 0; amHour <= 11; amHour++) {
+                expect(DateUtils.convert24HourMeridiem(amHour, false)).to.equal(pmHours[amHour]);
+            }
+        });
+
+        it("returns given hour, if hour is PM and not toAm", () => {
+            for (let hour = 12; hour <= 23; hour++) {
+                expect(DateUtils.convert24HourMeridiem(hour, false)).to.equal(hour);
+            }
+        });
+
+        it("returns given hour in AM, if hour is PM and toAm", () => {
+            const pmHours = [];
+            for (let i = 12; i <= 23; i++) {
+                pmHours.push(i);
+            }
+            for (let amHour = 0; amHour <= 11; amHour++) {
+                expect(DateUtils.convert24HourMeridiem(pmHours[amHour], true)).to.equal(amHour);
+            }
+        });
+
+        it("throws an error only for invalid hours", () => {
+            expect(() => DateUtils.convert24HourMeridiem(-1, true)).to.throw();
+            expect(() => DateUtils.convert24HourMeridiem(24, true)).to.throw();
+            expect(() => DateUtils.convert24HourMeridiem(0, true)).to.not.throw();
+            expect(() => DateUtils.convert24HourMeridiem(23, true)).to.not.throw();
+        });
+    });
+
+    describe("getIsPmFrom24Hour", () => {
+        it("returns true, if hour (in 24 range) is PM", () => {
+            expect(DateUtils.getIsPmFrom24Hour(12)).to.equal(true);
+            expect(DateUtils.getIsPmFrom24Hour(23)).to.equal(true);
+
+            for (let hour = 12; hour <= 23; hour++) {
+                expect(DateUtils.getIsPmFrom24Hour(hour)).to.equal(true);
+            }
```

Comment:
so you added some explicit values, now it's time to remove the `for` loop.
(this applies to several tests here)